sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 ## Release History
 
-## 4.12.0-beta.1 (Unreleased)
+### 4.12.0 (2021-02-09)
+#### New Features
+* Added connection endpoint rediscovery feature to help reduce and spread-out high latency spikes.
+* Added changeFeed pull model beta API.
+* Added support for resuming query from a pre split continuation token after partition split.
+* Optimized query execution time by caching query plan.
```

Comment:
Lets please expand more on which type of queries are handled.

Reply:
hmm how about Optimized query execution time by caching query plan  for single partition queries with filters and orderby

Reply:
updated in a separate PR